### ISSUE TITLE
Item: Humbert's Sword (3693) incorrect drop rate?

### ISSUE THREAD

**Issue by wowquester** [azerothcore/azerothcore-wotlk#16704]
### Current Behaviour

The current drop rate for Humbert's Sword (3693) (for quest 547) from Dun Garok dwarfs (and all related creatures see wiki) is 100%.

### Expected Blizzlike Behaviour

Shouldn't be 100%, even the quest says "...So many dwarves there...it will be quite difficult to find the dirty little thief who stole it."
Need sniff. 

### Source

https://www.youtube.com/watch?v=6qaFdQsL4aM
https://www.wowhead.com/wotlk/item=3693/humberts-sword#objective-of

### Steps to reproduce the problem

Horde char
.add q 547
.go xyz -1257.947998 -1191.317627 38.944157
Kill any dwarf

### AC rev. hash/commit

bda9b404038e

### Operating system

Win11x64

### Custom changes or Modules

None

**Comment by elthehablo** [comment]
Loot chances cannot be sniffed, as stated in an earlier issue made by you.

**Comment by elthehablo** [comment]
I do agree that the drop chance shouldn't be 100%, and it is 100% as seen in the DB.

**Comment by neifion-00000000** [comment]
> Loot chances cannot be sniffed, as stated in an earlier issue made by you.

this is not entirely true, sniffing is a log so to speak, but there is not a parsing method to easily decipher the sniffs.  I have sniffed drop rates via constantly deleting the quest items before for example and thus having a "clean" sniff of the drop rate, but in a pinch we can determine rates, it just takes time, though 100% rates are easiest to prove or deny.  

There does seem to be something peculiar about this specific quest though if the comments are to be believed.

**Comment by elthehablo** [comment]
I agree with the issue though. I distinctly remember doing this quest on classic and taking a lot of time to get it

**Comment by elthehablo** [comment]
> UPDATE `acore_world`.`creature_loot_template` SET `Chance`=10 WHERE  `Entry`=2344 AND `Item`=3693 AND `Reference`=0 AND `GroupId`=0;
> UPDATE `acore_world`.`creature_loot_template` SET `Chance`=10 WHERE  `Entry`=2345 AND `Item`=3693 AND `Reference`=0 AND `GroupId`=0;
> UPDATE `acore_world`.`creature_loot_template` SET `Chance`=10 WHERE  `Entry`=2346 AND `Item`=3693 AND `Reference`=0 AND `GroupId`=0;
> UPDATE `acore_world`.`creature_loot_template` SET `Chance`=10 WHERE  `Entry`=14275 AND `Item`=3693 AND `Reference`=0 AND `GroupId`=0;

Based on?
